examples/social-network/websocket-server/functions/onLikeDislike.js

```diff
@@ -33,6 +33,18 @@ export default async function (event) {
         },
       }
     }],
+    reply: [
+      {
+        channel: '/',
+        payload: {
+          type: 'likes_count_updated',
+          data: {
+            postId,
+            totalCount
+          }
+        }
+      }
+    ],
```

Comment:
@fmvilas when I trigger `reply` event I still get `['send']` from `message.eventName()`. 
Shouldn't it be `['reply']` when `message.reply` function is called. 
https://github.com/asyncapi/glee/blob/2b2fb5455c034f73851afe9781d73c9116084155/src/lib/functions.ts#L72

Reply:
Where do you get `['send']` from `message.eventName()`?

Reply:
yeah in `message.eventName()`, I am trying to differentiate between two different events but I am getting the same parameter.

Reply:
Oh sorry I mean `message.eventNames()` and I am accessing it in `message` object in the `logOutBoundMessage`. 

> However, when we're sending to a WebSockets server, it should say "was broadcasted" unless it was a reply to a specific WebSockets client. 

For this, I am trying to access the event type of the messages, Now `broadcast` is deprecated and `send` event has `broadcast:true` so I guess we can check the event that way.

Reply:
> Oh sorry I mean message.eventNames()

But where is `message.eventNames()` defined? 🤔 I don't think it's something from our code, is it? I think what you're using is this [`eventNames`](https://nodejs.org/api/events.html#emittereventnames) which I don't think will be useful at all for you. This is something from Node.js EventEmitter so I honestly have no idea what value you'll get there.